lib/sendMetrics/index.js

```diff
@@ -0,0 +1,34 @@
+const SDC = require('@fiverr/statsd-client');
+
+function send({ statsdClient, dataFromApi }) {
+    const statsd = new SDC({ host: statsdClient.host, port: statsdClient.port });
+    const data = prepareDataFromApi(dataFromApi);
+    data.forEach((permutation) => {
+        statsd.gauge(buildStatsdMetric(permutation.page, permutation.strategy, permutation.category), permutation.score);
+    });
+}
+
+function prepareDataFromApi(dataFromApi) {
```

Comment:
This whole function can be one long, fun, reducer. If you're into such things. And with property destructors, it can have 0 code blocks.
```js
const prepareDataFromApi = (dataFromApi) => dataFromApi.map(
    (arr) => arr.map(
        ({ pageName, strategy }) => url.details.map(
            ({ name: category, score }) => ({
                page: pageName.split(' ').join('').toLowerCase(),
                strategy,
                category,
                score
            })
        )
    ).flat()
).flat();
```

I think, this is fun only for me